scripts/deploy-rpc-playbooks.sh

```diff
@@ -36,13 +36,15 @@ if [[ "${DEPLOY_ELK}" == "yes" ]]; then
   run_ansible setup-logging.yml
 fi
 
+# Download the latest release of rpc-maas
+run_ansible maas-get.yml
```

Comment:
nit: I'm not sure why this should be outside the ``DEPLOY_MAAS`` conditional below, but it doesn't hurt.